inst/include/Rcpp/macros/mask.h

```diff
@@ -22,7 +22,7 @@
 
 #ifndef RCPP_NO_MASK_RF_ERROR
 #define Rf_error(...) \
-    _Pragma("message \"Use of Rf_error() instead of Rcpp::stop(). Calls \
+    _Pragma("GCC warning \"Use of Rf_error() instead of Rcpp::stop(). Calls \
```

Comment:
I don't think it matters either way but in RcppArmadillo I used `#pragma message`.  Kurt was able to filter these with a heads-up.  That would be one softer than a warning.